src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -457,4 +429,13 @@ private function findAsset(string $path): ?MappedAsset
 
         return $this->assetMapper->getAssetFromSourcePath($path);
     }
+
+    private function createMissingImportMapAssetException(ImportMapEntry $entry): \InvalidArgumentException
```

Comment:
should return a LogicException I suppose, that's how we report such errors usually IIRC - let's also add the exception class to the component's namespace with its interface?